docs/dapps/build-dapp-from-scratch.md

```diff
@@ -0,0 +1,548 @@
+---
+sidebar_position: 50
+title: Build dApp from scratch
+sidebar_label: Build dApp from scratch
+---
+
+import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+
+<UntranslatedPageText />
+
+#Dapps Getting started
```

Comment:
We could remove this since there is a tile for the page already? (We don't use such titles in other pages)

Reply:
This is the layout right now:
<img width="293" alt="Screenshot 2023-03-20 at 09 56 40" src="https://user-images.githubusercontent.com/295279/226291582-e6f7be19-68e1-4429-93f0-23a1610e5ac8.png">